src/jekyll/running-on-kubernetes.md

```diff
@@ -290,6 +290,55 @@ the command may then look like the following:
 
 ## Advanced
 
+### Configuring Kubernetes Roles and Service Accounts
+
+#### Driver
+
+The Spark driver pod uses a Kubernetes service account to access the Kubernetes API server to create and watch executor pods. The service account used by the driver pod must have the appropriate permission for the driver to be able to do its work. 
+
+Specifically, at minimum, the service account must have the [`edit`](https://kubernetes.io/docs/admin/authorization/rbac/#user-facing-roles) [`Role` or `ClusterRole`](https://kubernetes.io/docs/admin/authorization/rbac/#role-and-clusterrole) granted. By default, the driver pod is automatically assigned the `default` service account in the namespace specified by `--kubernetes-namespace`, if no service account is specified when the pod gets created. 
```

Comment:
Must have "edit" on pods and services?

Reply:
Having the default "edit" role is a bit more than we need right?

Reply:
Made it more specific that a role that allows creating pods and secrets is needed.